sdk/search/search-documents/README.md

```diff
@@ -19,7 +19,7 @@ Use the client library to:
 
 [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/search/search/) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/search-documents) |
-[API reference documentation](https://aka.ms/azsdk-js-search-ref-docs) |
+[API reference documentation](https://aka.ms/azsdk/js/docs/search) |
```

Comment:
I can't remember, did we decide to not include "ref" in the aka?

@kaerm

Reply:
I think we said to delete everything. Eventually azsdk/js/docs probably should lead to developer centers on docs so that might be a bit confusing if anyone cares about the link structure. Still, I can't think of a place where these would not lead to ref docs.

Reply:
@jongio I thought we agreed on either `ref` or `docs`. I can change them to `ref`! Would that be better?

Reply:
Ok I reached through teams and we've agreed on removing the docs part! On it!

Reply:
moved from docs/<service> to <service>/docs, as agreed upon over Teams